app/controllers/api/v1/memberships_controller.rb

```diff
@@ -4,9 +4,18 @@ class Api::V1::MembershipsController < Api::ApiController
   resource_actions :index, :show, :create, :update, :deactivate
   schema_type :strong_params
 
-  allowed_params :create, links: [:user, :user_group]
   allowed_params :update, :state
 
+  def create
```

Comment:
Obviously we'd extract this when we get more actions like this. For now I'm not wanting to over-engineer.